File: Languages/Kotlin/FizzBuzz.kt
fun fizzBuzzOf(input: Int) : String {
   var result : String = "";
   
   if(input % 3 == 0){
   	result += "Fizz"
   }
   
   if(input % 5 == 0){
   	result += "Buzz"
   }
   
   return if(!result.isNullOrEmpty()) result else input.toString()	
}

File: Go/fizzBuzz.go
package main

import (
	"fmt"
)

func main(){
	for i:=1;i<100; i++{
		if i % 3 == 0 && i % 5 != 0{
			
			fmt.Println("Fizz",i)
		}else if i % 3 != 0 && i % 5 == 0{
			fmt.Println("Buzz",i)
		}else{
			fmt.Println("FizzBuzz",i)
		}
	}
}